src/hooks/widgets/rss/useGetRssFeed.tsx

```diff
@@ -1,10 +1,10 @@
 import { useQuery } from '@tanstack/react-query';
 
-export const useGetRssFeed = (feedUrl: string) =>
+export const useGetRssFeed = (feedUrl: string, widgetId: string) =>
   useQuery({
     queryKey: ['rss-feed', feedUrl],
     queryFn: async () => {
-      const response = await fetch('/api/modules/rss');
+      const response = await fetch(`/api/modules/rss?widgetId=${widgetId}`);
```

Comment:
Isn't this required for more APIs ?

Reply:
As far as i've seen this should be done for all api's. I'll check if something is missing

Reply:
Yeah every api request depending on the widget id is done